include/swift/AST/DiagnosticsSIL.def

```diff
@@ -944,39 +944,25 @@ ERROR(regionbasedisolation_unknown_pattern, none,
       "pattern that the region based isolation checker does not understand how to check. Please file a bug",
       ())
 
-//===---
-// Old Transfer Non Sendable Diagnostics
-//
-
-ERROR(regionbasedisolation_transfer_yields_race_no_isolation, none,
-      "sending value of non-Sendable type %0 with later accesses risks causing data races",
-      (Type))
-ERROR(regionbasedisolation_transfer_yields_race_with_isolation, none,
-      "sending value of non-Sendable type %0 with later accesses from %1 context to %2 context risks causing data races",
-      (Type, ActorIsolation, ActorIsolation))
-ERROR(regionbasedisolation_isolated_capture_yields_race, none,
-      "%1 closure captures value of non-Sendable type %0 from %2 context; later accesses to value could race",
-      (Type, ActorIsolation, ActorIsolation))
-ERROR(regionbasedisolation_transfer_yields_race_stronglytransferred_binding, none,
-      "value of non-Sendable type %0 accessed after being transferred; later accesses could race",
-      (Type))
-ERROR(regionbasedisolation_arg_transferred, none,
-      "sending %0 value of type %1 with later accesses to %2 context risks causing data races",
-      (StringRef, Type, ActorIsolation))
-ERROR(regionbasedisolation_arg_passed_to_strongly_transferred_param, none,
-      "%0 value of type %1 passed as a strongly transferred parameter; later accesses could race",
-      (StringRef, Type))
-
-//===---
-// New Transfer Non Sendable Diagnostics
-//
-
 ERROR(regionbasedisolation_named_transfer_yields_race, none,
       "sending %0 risks causing data races",
       (Identifier))
 NOTE(regionbasedisolation_type_is_non_sendable, none,
       "%0 is a non-Sendable type",
       (Type))
+ERROR(regionbasedisolation_type_transfer_yields_race, none,
+      "sending value of non-Sendable type %0 risks causing data races",
+      (Type))
+
+NOTE(regionbasedisolation_type_use_after_transfer, none,
+      "sending value of non-Sendable type %0 to %1 callee risks causing data races between %1 and local %2 uses",
```

Comment:
Can you please avoid introducing new uses of the word `callee` in user-facing diagnostics? It doesn't look like this language was used in the old set of diagnostics.

Reply:
I can change it to use the word function.